cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/support/HardTimeoutExpirationPolicy.java

```diff
@@ -63,7 +63,7 @@ public void init() {
     @Override
     public boolean isExpired(final TicketState ticketState) {
         return (ticketState == null) || ticketState.getCreationTime()
-          .plus(this.timeToKillInMilliSeconds, ChronoUnit.MILLIS).isAfter(ZonedDateTime.now(ZoneOffset.UTC));
+          .plus(this.timeToKillInMilliSeconds, ChronoUnit.MILLIS).isBefore(ZonedDateTime.now(ZoneOffset.UTC));
```

Comment:
So, I am being extra thick. What does this say in plain english? `If the creationTime + margin is before now, ticket is expired?`